docs/source/installation.rst

```diff
@@ -4,76 +4,70 @@ Installation
 
-
-The recommended way to install the Debug Toolbar is via pip_::
-    
     $ pip install django-glitter
 
-If you aren't familiar with pip, you may also obtain a copy of the
-``glitter`` directory and add it to your Python path.
-
 .. _pip: http://www.pip-installer.org/
 
 
-Installation
-------------
+settings.py
+===========
 
+INSTALLED_APPS
+--------------
 
-You will need to add the ``glitter`` application to the `INSTALLED_APPS`
-setting of your Django project ``settings.py`` file.::
+You'll need to add ``glitter`` to `INSTALLED_APPS`, as well as any app dependencies, or additional
+blocks which are included with Glitter as Django apps::
 
     INSTALLED_APPS = (
         #...
+
+        # Glitter, and recommended packages
         'glitter',
         'glitter.pages',
-        #...
-    )
-
+        'glitter.assets',
 
-.. note::
+        # Standard blocks
+        'glitter.blocks.html',
+        'glitter.blocks.image',
+        'glitter.blocks.redactor',
 
-    We need to mention here about our required packages ``django-mptt``,
-    ``django-mptt-admin``, ``sorl-thumbnail`` and hint users to add in
-    `INSTALLED_APPS`.
+        # Glitter pages dependencies
+        'mptt',
+        'django_mptt_admin',
+        'sorl.thumbnail',
 
-.. note::
-    
-    Explain diffences between ``glitter`` and ``glitter.pages``
+        #...
+    )
 
 
-URLconf
--------
+MIDDLEWARE_CLASSES
+------------------
 
-Add the ``glitter`` URLs to your project's URLconf as follows::
+Glitter uses ``PageFallbackMiddleware`` as a 404 page handler to render a page if no other view
+handles the request, and ``ExceptionMiddleware`` to handle exceptions generated by Glitter views.
 
-    from glitter import block_admin
+You'll need to add these lines to the **bottom** of ``MIDDLEWARE_CLASSES``::
 
-    urlpatterns = [
-        #...
-        url(r'^blockadmin/', include(block_admin.site.urls)),
+    MIDDLEWARE_CLASSES = [
         #...
+        'glitter.pages.middleware.PageFallbackMiddleware',
+        'glitter.middleware.ExceptionMiddleware',
     ]
 
 
-Middleware
-----------
-
-Each time any Django application raises a 404 error, this
-``glitter.pages.middleware.PageFallbackMiddleware`` middleware checks the
-glitter pages database for the requested URL as a last resort
+urls.py
+=======
 
-The middleware ``glitter.middleware.ExceptionMiddleware`` is handles              
-exceptions if the object doesn't have the current version or hasn't been
-published it prompts the user to create a new page it also deals with blocks
-which raise exception ``GlitterRedirectException``::
+The Glitter block admin needs registering in your ``urls.py``::
 
+    from glitter.blockadmin import blocks
 
-    MIDDLEWARE_CLASSES = [
+    urlpatterns = [
         #...
-        'glitter.pages.middleware.PageFallbackMiddleware',
-        'glitter.middleware.ExceptionMiddleware',
+        url(r'^blockadmin/', include(blocks.site.urls)),
```

Comment:
Nope!

For the last released version, you would.

However the work you've done has moved this to `glitter.blockadmin.blocks`, so it's `blocks.sites.urls`.